ert_shared/libres_facade.py

```diff
@@ -122,7 +122,7 @@ def gather_gen_kw_data(self, case, key):
         """ :rtype: pandas.DataFrame """
         data = GenKwCollector.loadAllGenKwData(self._enkf_main, case, [key])
         if key in data:
-            return data[key].dropna()
+            return data[key].to_frame().dropna()
```

Comment:
This part, although it probably should be added to keep consistent return data structure for all `get` functions, it does cause a bug.

https://github.com/equinor/ert/issues/720

Reply:
Agreed!